spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/src/main/resources/application.yml

```diff
@@ -0,0 +1,20 @@
+---
+cloudfoundry:
```

Comment:
I am expecting these settings to be in the manifest of the application/service that is deployed on CF because they need to be supplied from outside of the app somehow.  Most of the oauth2 parameters are set this way, so we need to set them for the entire CF app.  I put them in my own application.yml so that it is clear what is needed. The names will change, doubtless, although the oauth2 stuff cannot (I don't think).